realm/realm-library/src/main/java/io/realm/DynamicRealm.java

```diff
@@ -71,6 +71,26 @@ public static DynamicRealm getInstance(RealmConfiguration configuration) {
     }
 
     /**
+     * The creation of the first Realm instance for per {@link RealmConfiguration} in the process takes time to do
```

Comment:
The creation of the first Realm instance per {@link RealmConfiguration} in a process can take some time as all initialization code need to run at that point (Setting up the Realm, validating schemas and creating initial data).